src/python/qepsi4/_psi4_test.py

```diff
@@ -50,20 +125,55 @@ def test_run_psi4():
     # For this system, the CCSD energy should be exact.
     assert math.isclose(energy, results_cisd["energy"], rel_tol=1e-7)
 
-def test_get_rdms_from_psi4():
+def test_run_psi4_1():
+
+    results, hamiltonian = run_psi4(hydrogen_geometry, save_hamiltonian=True, n_active_extract=1)
+    assert math.isclose(results["energy"], -0.8544322638069642)
+    assert results["n_alpha"] == 1
+    assert results["n_beta"] == 1
+    assert results["n_mo"] == 2
+    assert results["n_frozen_core"] == 0
+    assert results["n_frozen_valence"] == 0
+
+    assert hamiltonian.n_qubits == 2
+    qubit_operator = qubit_operator_sparse(jordan_wigner(hamiltonian))
+    energy, state = jw_get_ground_state_at_particle_number(qubit_operator, 2) # This should give RHF energy
+
+    assert math.isclose(energy, results["energy"], rel_tol=1e-3) # Note: this test only passes if rel_tol=1e-3 or lower; we should investigate this
+
+def test_run_psi4_2():
+
+    results, hamiltonian = run_psi4(hydrogen_geometry, save_hamiltonian=True, n_active_extract=1, n_occupied_extract=0)
+    assert math.isclose(results["energy"], -0.8544322638069642)
+    assert results["n_alpha"] == 1
+    assert results["n_beta"] == 1
+    assert results["n_mo"] == 2
+    assert results["n_frozen_core"] == 0
+    assert results["n_frozen_valence"] == 0
+
+    assert hamiltonian.n_qubits == 2
+    qubit_operator = qubit_operator_sparse(jordan_wigner(hamiltonian))
+    energy, state = jw_get_ground_state_at_particle_number(qubit_operator, 0) # This should give RHF energy in the 0-particle sector of the Fock space
+
+    assert math.isclose(energy, results["energy"], rel_tol=1e-3) # Note: same as the previous test; it is not affected by EQ_TOLERANCE and could be due to a bug in the OpenFermion
```

Comment:
Do we have a tracker story to investigate this?
Could it be because Psi4 computes the energy with the density fitting approximation?

Reply:
Hat tip to you, Jerome! scf_type pk fixes the problem! (Idk if I should introduce it to the code though since other tests were performed under the assumption of scf_type df and will fail if I hardcode pk)

Reply:
Ideally, in the future we would reinstate the `extra_commands` option that allowed to specify arbitrary option for Psi4. For now I think we can just leave these tests this way... I'll make a little tracker story to add this command as a separate PR.

Reply:
I think @max-radin already did that by introducing the `options` parameter that allows passing a dictionary of Psi4 options to `run_psi4` in a workflow. It was merged to the master last week.